tests/sherlock_interactives.py

```diff
@@ -7,7 +7,7 @@ class Interactives:
     def run_cli(args:str = "") -> str:
         """Pass arguments to Sherlock as a normal user on the command line"""
         # Adapt for platform differences (Windows likes to be special)
-        if platform.system == "Windows":
+        if platform.system() == "Windows":
             command:str = f"py -m sherlock {args}"
```

Comment:
With your fix, this incorrect code is actually running now.

```suggestion
            command:str = f"py -m sherlock_project {args}"
```

Windows is a pain. Always has to be special. Let's give this a try.